src/transform/volume.rs

```diff
@@ -429,6 +484,46 @@ mod tests {
         assert_eq!(images.len() as u32, expected_number_of_frames);
     }
 
+    #[rstest]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::Keep(KeepVolume), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::CentralSlice(CentralSlice), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::MaxIntensity(MaxIntensity { skip_start: 0, skip_end: 0 }), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::Interpolate(InterpolateVolume { target_frames: 32 }), 1)]
+    fn test_decode_volume_with_options(
+        #[case] dicom_file_path: &str,
+        #[case] volume_handler: VolumeHandler,
+        #[case] expected_number_of_frames: u32,
+    ) {
+        let dicom = dicom_test_files::path(dicom_file_path).unwrap();
+        let dicom = open_file(&dicom).unwrap();
+        let mut options = ConvertOptions::default();
+        options.voi_lut = VoiLutOption::Normalize;
+        let images = volume_handler
+            .decode_volume_with_options(&dicom, &options)
+            .unwrap();
+        assert_eq!(images.len() as u32, expected_number_of_frames);
+    }
+
+    #[rstest]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::Keep(KeepVolume), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::CentralSlice(CentralSlice), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::MaxIntensity(MaxIntensity { skip_start: 0, skip_end: 0 }), 1)]
+    #[case("pydicom/CT_small.dcm", VolumeHandler::Interpolate(InterpolateVolume { target_frames: 32 }), 1)]
+    fn test_par_decode_volume_with_options(
```

Comment:
Since this test is nearly identical to the previous test, maybe we could leverage `#values[...]` to run all desired test combinations?
```rust
    fn test_decode_volume_with_options(
        #[case] dicom_file_path: &str,
        #[case] volume_handler: VolumeHandler,
        #[case] expected_number_of_frames: u32,
        #[values(true, false)]
        parallel: bool,
    ) {
    ...
    let images = match parallel { ...
```